lib/DBDefs/Default.pm

```diff
@@ -271,6 +271,15 @@ sub CACHE_MANAGER_OPTIONS {
     return \%CACHE_MANAGER_OPTIONS
 }
 
+# Sets the TTL for entities stored in memcached, in seconds. On slave servers,
+# this is set to 1 hour by default, to mitigate MBS-8726. On standalone
+# servers, this is set to 0 (meaning no expiration is set), because cache
+# invalidation is already handled by the server in that case.
+sub ENTITY_CACHE_TTL {
+    return 3600 if shift->REPLICATION_TYPE == RT_SLAVE;
+    return 0;
+}
```

Comment:
Yeah, I had checked before if `Cache::Memcached::Fast` supported `default_expires`. Certainly confusing. Like I mentioned below, I think it makes sense to just remove the in-memory cache in a later PR. I'm afraid that setting an expiration of "0" on things in the in-memory cache would cause some items to never be invalidated in some now-known cases.